src/styles/_ibm-cookies.scss

```
@@ -0,0 +1,9 @@
+.truste_cookie_prefs a {
+  font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif !important;
```

It'll load the fallback font file unless we use the variable font family

```suggestion
  font-family: 'IBM Plex Sans VF', 'Helvetica Neue', Arial, sans-serif !important;
```